addons/mail/static/src/discuss/call/common/call_actions.js

```diff
@@ -0,0 +1,35 @@
+import { registry } from "@web/core/registry";
+
+export const callActionsRegistry = registry.category("discuss.call/actions");
+
+callActionsRegistry
+    .add("mute", {
+        isActive(session) {
+            return session?.isSelfMuted;
+        },
+        icon: "fa-microphone-slash",
+    })
+    .add("deafen", {
+        isActive(session) {
+            return session?.isDeaf;
+        },
+        icon: "fa-deaf",
+    })
+    .add("raise-hand", {
+        isActive(session) {
+            return session?.raisingHand;
+        },
+        icon: "fa-hand-paper-o",
+    })
+    .add("camera-on", {
+        isActive(session) {
+            return session?.isCameraOn;
+        },
+        icon: "fa-video-camera",
+    })
+    .add("share-screen", {
+        isActive(session) {
+            return session?.isScreenSharingOn;
+        },
+        icon: "fa-desktop",
+    });
```

Comment:
The purpose of this call action registry is to be reused, so that call actions data are shared in different parts of UI.

Call action list has custom template with each action. I'd expect to simply reuse the registry.
Each item can have more contextual data needed specifically for call action list. Notably active / inactive styling and titles. All of them can be in registry too.